module/VuFind/src/VuFind/Search/Factory/AbstractSolrBackendFactory.php

```diff
@@ -183,6 +206,69 @@ public function __invoke(ContainerInterface $sm, $name, array $options = null)
         return $backend;
     }
 
+    /**
+     * Return an ordered array of configurations to check for index configurations.
+     *
+     * @return string[]
+     */
+    protected function getIndexConfigFallbackPath(): array
```

Comment:
I'd rename this to something like `getIndexConfigPaths`. Current name sounds like it just returns a fallback instead of the all.

Reply:
I changed it to `getPrioritizedConfigsForIndexSettings` -- long, but more accurate.